tsconfig.build.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "./tsconfig.json",
-  "exclude": ["node_modules", "**/*.spec.ts", "**/*.test.ts", "example"],
+  "exclude": ["node_modules", "example", "test/**/*.ts", "**/*.spec.ts", "**/*.test.ts"],
```

Comment:
I'd exclude the whole `test` folder, since we might have extra non TS files in the future.